registry/modules/shared/utils/response.ts

```diff
@@ -0,0 +1,10 @@
+import { ResponseError } from "./errors/ResponseError.js";
+
+/**
+ * Standardized response format
+ * @param {T} data optional parameters to return.
+ * @param {ResponseError | Error} error the success or error message.
+ */
+export const response = <T>(data?: T, error?: ResponseError | Error) => {
+  return { data, error: { name: error?.name, message: error?.message } };
+};
```

Comment:
Looks like the response would always contain the error key.
```suggestion
export const response = <T>(data?: T, error?: ResponseError | Error) => {
  if(error !== undefined){
      return { data, error: { name: error?.name, message: error?.message } };
  }
  return { data };
};
```

Maybe we also dont always want to have the data key incase data is undefined.

Reply:
Great catch, `data` key shouldn't show if its undefined so it will return an empty object.